### PR TITLE
feat: adding collection for open sources from rwanda

### PR DESCRIPTION
**Pull request by FADHILI-Josue**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I am not the sole author or employee of a company who created either the topic I am modifying/adding or the collection entry I am modifying/adding.

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> I made a new collection named "Made In Rwanda" to help exploration of open sources made in Rwanda so that other developers from Rwanda can showcase their excellent work by adding their work in this made in Rwanda collection

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by tomthorogood** [comment]
@FADHILI-Josue thanks so much for your contribution to GitHub Explore! ✨  Could you please make sure to check the box above that says:

> I am not the sole author or employee of a company who created either the topic I am modifying/adding or the collection entry I am modifying/adding.

It also appears one of the repositories you linked has been renamed, so must be fixed before we can merge in your changes.

File: collections/made-in-rwanda/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ - pacifiquem/awesome-go
+ - mutesa-Cedric/react-swift-reveal
+ - pacifiquem/lepper
+ - pacifiquem/lin
+  
+display_name: Made in Rwanda
+created_by: FADHILI Josue
+image: made-in-rwanda.png
+---
+Open source projects built in or receiving significant contributions from rwanda :rwanda:
```
Comment (kenyonj) [inline collections/made-in-rwanda/index.md:15]:
```suggestion
Open source projects built in or receiving significant contributions from Rwanda :rwanda:
```

**Review by kenyonj — CHANGES_REQUESTED** [review]

File: collections/made-in-rwanda/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+items:
+ - kin-lang/kin
+ - veritem/eslint-plugin-vitest
+ - divinrkz/swaggiffy
+ - pacifiquem/awesome-go
+ - mutesa-Cedric/react-swift-reveal
```
Comment (kenyonj) [inline collections/made-in-rwanda/index.md:7]:
```suggestion
 - Mutesa-Cedric/react-swift-reveal
```

**Review by kenyonj — APPROVED** [review]
Looks great. Thanks for your contribution!

File: collections/made-in-rwanda/index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+ - kin-lang/kin
+ - veritem/eslint-plugin-vitest
+ - divinrkz/swaggiffy
+ - pacifiquem/awesome-go
+ - Mutesa-Cedric/react-swift-reveal
+ - pacifiquem/lepper
+ - pacifiquem/lin
+  
+display_name: Made in Rwanda
+created_by: FADHILI Josue
```
Comment (kenyonj) [inline collections/made-in-rwanda/index.md:12]:
```suggestion
created_by: FADHILI Josue
```

**Pull request merged**